<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>website_load</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>400</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>351</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLineEdit" name="target_lineEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="placeholderText">
            <string>Enter target url here..</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_readurl_from_file">
         <property name="text">
          <string>get urls from file &quot;urls.txt&quot;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="checkBox_read_proxies">
           <property name="text">
            <string>get proxies from file &quot;proxies.txt&quot; (if exists)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>368</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Referrer:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="referrer"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Threads:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_Threads">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>2000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>1900</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Request Types:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_Head">
           <property name="text">
            <string>HEAD</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_Post">
           <property name="text">
            <string>POST</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_Get">
           <property name="text">
            <string>GET</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Timeout</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>from:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="timeout_min">
           <property name="text">
            <string>30</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="timeout_max">
           <property name="text">
            <string>30</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="wait_for_response">
           <property name="text">
            <string>Wait for response</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="append_random_chars">
           <property name="text">
            <string>Append random characters to url</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>startButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
